systers_portal/systers_portal/settings/runner.py

```diff
@@ -0,0 +1,29 @@
+class PytestTestRunner(object):
+    """Runs pytest to discover and run tests."""
+
+    def __init__(self, verbosity=1, failfast=False, keepdb=False, **kwargs):
+        self.verbosity = verbosity
+        self.failfast = failfast
+        self.keepdb = keepdb
+
+    def run_tests(self, test_labels):
+        """Run pytest and return the exitcode.
+
+        It translates some of Django's test command option to pytest's.
+        """
+        import pytest
+
+        argv = []
+        if self.verbosity == 0:
+            argv.append('--quiet')
+        if self.verbosity == 2:
```

Comment:
Out of curiosity, what would verbosity=1 mean?

Reply:
Hello @nida by default the value is one. 
https://github.com/Fenn-CS/portal/blob/5f43573a75e81075906f83b17df76ef6b887f34a/systers_portal/systers_portal/settings/runner.py#L4

And on one, all the "verbose" reports irrespective will be displayed on shell.

Reply:
Makes sense. Thanks for the explanation